runtime/runtime.go

```diff
@@ -92,7 +60,7 @@ type Runtime interface {
 	// ParseAndCheckProgram parses and checks the given code without executing the program.
 	//
 	// This function returns an error if the program contains any syntax or semantic errors.
-	ParseAndCheckProgram(source []byte, context Context) (*sema.Checker, error)
+	ParseAndCheckProgram(source []byte, context Context) (*interpreter.Program, error)
```

Comment:
We might need to update consensus nodes for this change

Reply:
This function is neither used anywhere in Cadence, not anywhere in Flow Go. I think it used to be used maybe at some point, or just exists for convenience reasons.